packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -0,0 +1,94 @@
+import { getDeployedProxiedContract } from '@celo/protocol/lib/web3-utils'
+import {
+  GoldTokenInstance,
+  RegistryInstance,
+  ReleaseGoldContract,
+  ReleaseGoldMultiSigContract,
+  ReleaseGoldMultiSigProxyContract,
+  ReleaseGoldProxyContract,
+} from 'types'
+import fs = require('fs')
+
+module.exports = async (callback: (error?: any) => number) => {
+  try {
+    const argv = require('minimist')(process.argv.slice(2), {
+      string: ['network', 'grants'],
+    })
+    const registry = await getDeployedProxiedContract<RegistryInstance>('Registry', artifacts)
+    const goldToken = await getDeployedProxiedContract<GoldTokenInstance>('GoldToken', artifacts)
+    const ReleaseGoldMultiSig: ReleaseGoldMultiSigContract = artifacts.require(
+      'ReleaseGoldMultiSig'
+    )
+    const ReleaseGoldMultiSigProxy: ReleaseGoldMultiSigProxyContract = artifacts.require(
+      'ReleaseGoldMultiSigProxy'
+    )
+    const ReleaseGold: ReleaseGoldContract = artifacts.require('ReleaseGold')
+    const ReleaseGoldProxy: ReleaseGoldProxyContract = artifacts.require('ReleaseGoldProxy')
+    const releases = []
+    const handleJSONFile = async (err, data) => {
+      if (err) {
+        throw err
+      }
+      for (const releaseGoldConfig of JSON.parse(data)) {
+        const releaseStartTime = new Date(releaseGoldConfig.releaseStartTime)
+        const releaseCliffTime = releaseGoldConfig.releaseCliffTime
+        const numReleasePeriods = releaseGoldConfig.numReleasePeriods
+        const releasePeriod = releaseGoldConfig.releasePeriod
+        const amountReleasedPerPeriod = releaseGoldConfig.amountReleasedPerPeriod
+        const revocable = releaseGoldConfig.revocable
+        const beneficiary = releaseGoldConfig.beneficiary
+        const releaseOwner = releaseGoldConfig.releaseOwner
+        const refundAddress = releaseGoldConfig.refundAddress
+        const subjectToLiquidityProvision = releaseGoldConfig.subjectToLiquidityProvision
+        const initialDistributionPercentage = releaseGoldConfig.initialDistributionPercentage
+        const canValidate = releaseGoldConfig.canValidate
+        const canVote = releaseGoldConfig.canVote
+        const releaseGoldInstance = await ReleaseGold.new(
+          releaseStartTime.getTime() / 1000,
+          releaseCliffTime,
+          numReleasePeriods,
+          releasePeriod,
+          amountReleasedPerPeriod,
+          revocable,
+          beneficiary,
+          releaseOwner,
+          refundAddress,
+          subjectToLiquidityProvision,
+          initialDistributionPercentage,
+          canValidate,
+          canVote,
+          registry.address
+        )
+        const releaseGoldProxyInstance = await ReleaseGoldProxy.new()
+        const releaseGoldMultiSigProxyInstance = await ReleaseGoldMultiSigProxy.new()
+        await goldToken.transfer(
```

Comment:
Should we only transfer after the multisig owner is setup?

Reply:
+1

Reply:
Is this handled now? Not sure how the wording is meant

Reply:
Actually maybe this is okay, since the proxy will be owned by the deployer